src/main/java/org/sagebionetworks/web/client/presenter/users/PasswordResetPresenter.java

```diff
@@ -2,180 +2,28 @@
 
 import com.google.gwt.activity.shared.AbstractActivity;
 import com.google.gwt.event.shared.EventBus;
-import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.Window;
 import com.google.gwt.user.client.ui.AcceptsOneWidget;
 import com.google.inject.Inject;
-import org.sagebionetworks.repo.model.ErrorResponseCode;
-import org.sagebionetworks.repo.model.UserProfile;
-import org.sagebionetworks.repo.model.auth.ChangePasswordWithCurrentPassword;
-import org.sagebionetworks.web.client.ClientProperties;
-import org.sagebionetworks.web.client.DisplayConstants;
-import org.sagebionetworks.web.client.DisplayUtils;
-import org.sagebionetworks.web.client.GlobalApplicationState;
-import org.sagebionetworks.web.client.SynapseJavascriptClient;
-import org.sagebionetworks.web.client.place.LoginPlace;
 import org.sagebionetworks.web.client.place.users.PasswordReset;
 import org.sagebionetworks.web.client.presenter.Presenter;
-import org.sagebionetworks.web.client.security.AuthenticationController;
-import org.sagebionetworks.web.client.view.users.PasswordResetView;
-import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
-import org.sagebionetworks.web.shared.exceptions.NotFoundException;
 
-@SuppressWarnings("unused")
 public class PasswordResetPresenter
   extends AbstractActivity
-  implements PasswordResetView.Presenter, Presenter<PasswordReset> {
+  implements Presenter<PasswordReset> {
 
+  private static final String ONE_SAGE_RESET_PASSWORD_URL =
+    "https://accounts.sagebionetworks.synapse.org/resetPassword";
```

Comment:
TODO: figure out how to maintain the current stack across a session

Reply:
This should set the appId to "synapse.org" (in case it has been customized for some other portal experience)